cli/README.md

```diff
@@ -0,0 +1,189 @@
+# NEO-GO client
+
+The Neo-Go client is used to run node, create/compile/deploy/invoke/debug smart contracts, run vm and operate with the wallet. 
+
+## Running node
+
+To start Neo Go node it's required to run 
+[neo-privatenet](https://hub.docker.com/r/cityofzion/neo-privatenet/) Docker image on your machine.
+
+To run node you can use
+```
+make run
+```
+
+or run the binary directly:
+
+```
+./bin/neo-go node
+```
+
+By default the node will run on the `private network`, to change this use additional network flag:
+
+```
+./bin/neo-go node --mainnet
+```
+
+#### Available network flags
+- `--mainnet, -m`
+- `--privnet, -p`
+- `--testnet, -t`
+
+#### Configuration path
+
+If you want to use particular configuration path you can set it up by
+
+`./bin/neo-go node --config-path /user/yourConfigPath`
+
+## Configuration
+
+All config files are located in /neo-go/config and they are differentiated according to the network type:
```

Comment:
`/neo-go/config` doesn't look right to me. At the moment it either gonna be local `./config`, or absolute `/config` for Docker image.